table/server/underdb/glue/src/main/java/alluxio/table/under/glue/Property.java

```diff
@@ -142,11 +142,17 @@ public static void unregister(Property Property) {
           .setDescription("The maximum number of connection to glue metastore.")
           .build();
 
+  public static final Property MAX_GLUE_FETCH_PARTITIONS =
+      new Builder(Name.MAX_GLUE_FETCH_PARTITIONS)
+          .setDefaultValue("256")
```

Comment:
Is this large enough? Do you know if we can make this larger while still allowing it to return the correct amount?

Reply:
This value means the maximum number of partitions to return in a single response, from 1 to 1000, 256 should large enough for most cases. I will update the explanation of this property.

Reply:
If we set it to 1000, is it slow? I think setting it larger would be better.

Reply:
I don't know the performance difference with an accurate number, but it should has little difference.

Reply:
I changed the default value to 512, which should more than enough for most cases.

Reply:
I changed it to 512, it still only fetch 460 partitions every time, it would not reach the limitation.